packages/big-bang-core/README.md

```diff
@@ -8,7 +8,11 @@ This package deploys [Big Bang Core](https://repo1.dso.mil/platform-one/big-bang
 
 ## Known Issues
 
-- Big Bang requires an AMD64 system to deploy as Iron Bank does not yet support ARM.  You will need to deploy to a cluster that is running AMD64.  Specifically, M1 Apple computers are not supported locally and you will need to provision a remote cluster to work with Big Bang currently.
+Lots of new things
```

Comment:
Is "Lots of new things" meant as a known issue? Meant to capture the `bigbang` noun being new?